### DISCUSSION TITLE
Enterprise Policies

### DISCUSSION THREAD

**Discussion by 11then** [sql-bi/Bravo#860] [General]
The registry documentation page mentions enterprise applied rules (e.g. telemetry) but is stored in the local PC registry.  How would that apply across the enterprise and not just that user on that machine?

**Comment by albertospelta** [comment]
Enterprise policies applied via the Windows registry are stored under the HKEY_LOCAL_MACHINE (HKLM) hive, which applies to the entire machine rather than individual users. This ensures that only users with administrative privileges can modify these settings, preventing unauthorized changes by standard users.

In an enterprise environment with Active Directory, Group Policies are deployed using administrative templates (.admx/.adml) and distributed to machines via the domain. These policies are automatically applied and enforced on client PCs, populating the relevant registry keys under HKLM. 

For further details you can see the docs here https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/group-policy/group-policy-overview
